tests/test_BukuCrypt.py

```diff
@@ -29,11 +28,9 @@ def test_encrypt_decrypt(tmpdir, filesize):
     with open(dbfile, 'wb') as fp:
         fp.write(content)
     assert os.stat(dbfile).st_size == filesize
-    with mock.patch('getpass.getpass', return_value='password'):
-        from buku import BukuCrypt
-        with pytest.raises(SystemExit):
-            BukuCrypt.encrypt_file(1, dbfile=dbfile)
-        BukuCrypt.decrypt_file(1, dbfile=dbfile)
+    from buku import BukuCrypt
+    BukuCrypt.encrypt_file(1, dbfile=dbfile, password='password')
+    BukuCrypt.decrypt_file(1, dbfile=dbfile, password='password')
```

Comment:
No need to mock `getpass()` now; `password` can be supplied directly.

Also `BukuCrypt.encrypt_file()` no longer attempts to halt the Python process on success.